api/src/services/eml-service.ts

```diff
@@ -120,14 +124,79 @@ export class EMLService extends DBService {
     return { organizationName: organizationName, onlineUrl: onlineUrl };
   }
 
+  /**
+   * Compares the submission `Boundary` spatial component against BCGW layers, and builds an `additionalMetadata`
+   * object containing the names of the matching layer features (ie: which regions does the submission boundary cross).
+   *
+   * @param {number} submissionId
+   * @param {string} datasetId
+   * @return {*}  {Promise<any>}
+   * @memberof EMLService
+   */
+  async getRegionAdditionalMetadataNode(submissionId: number, datasetId: string): Promise<any> {
+    const spatialService = new SpatialService(this.connection);
+
+    // Fetch the submission `Boundary` spatial component
+    const boundarySpatialComponent = await spatialService.getGeometryAsWktFromBoundarySpatialComponentBySubmissionId(
+      1 || submissionId,
```

Comment:
Was this meant to eventually become just `submissionId`? Currently this just resolves to `1`, right?

Reply:
Yeah that was just for testing, I've since pushed up the fix.